tests/pql/test_declarations.py

```diff
@@ -0,0 +1,9 @@
+from ats.pql.pql import evaluate_query
+
+
+def test_simple_stmt_declaration():
+    evaluate_query(
```

Comment:
poki co sprawdziles czy Twoj parser dziala. Sprawdz tez czy wynik jest dobry

Reply:
nie rozumiem jak to powinno być zrobione

Reply:
po prostu assertami sprawdzasz czy wszystko jest ok. np:

```
query = evaluate_query(...)
assert query.variables == ["s1"]
```

Reply:
ta już ogarnalem